app/lib/frontend/handlers.dart

```diff
@@ -82,8 +82,13 @@ Future<shelf.Response> appHandler(
   } else if (path.startsWith(staticUrls.staticPath)) {
     return _staticsHandler(request);
   } else if (staticRootFiles.contains(path)) {
-    return _staticsHandler(request,
-        pathOverride: '${staticUrls.staticPath}$path');
+    print(path);
+    var pathOverride = '${staticUrls.staticPath}$path';
+    if (path == '/robots.txt' && !_isProd(request)) {
+      pathOverride = '${staticUrls.staticPath}/no_robots.txt';
```

Comment:
Could this be
```dart
return rejectRobotsHandler(request);
```
instead? (So we only have the reject robots.txt contents in one place.)